cognite/client/_api_client.py

```diff
@@ -636,7 +636,7 @@ def _raise_API_error(res: Response, payload: Dict):
             error_details["duplicated"] = duplicated
         error_details["headers"] = res.request.headers.copy()
         APIClient._sanitize_headers(error_details["headers"])
-        log.debug("HTTP Error %s %s %s: %s", code, res.request.method, res.request.url, msg, extra=error_details)
+        log.debug("HTTP Error {} {} {}: {}".format(code, res.request.method, res.request.url, msg, extra=error_details))
```

Comment:
I think log.debug supports sprintf statements, so maybe keep the original here

Reply:
Some lines below we have the following:
`log.debug("HTTP/{} {} {} {}".format(http_protocol_version, method, url, status_code), extra=extra)`